common/src/main/java/com/wynntils/wc/custom/item/WynnItemStack.java

```diff
@@ -46,7 +46,7 @@ public void addProperty(ItemProperty property) {
      */
     public <T> T getProperty(Class<T> propertyType) {
         for (ItemProperty property : properties) {
-            if (property.getClass().equals(propertyType)) return propertyType.cast(property);
```

Comment:
This looks like a merge error..?

Reply:
err, I don't think so? I saw the condition changed from main so I took that and combined it with the class cast()

Reply:
Agreed. Github presented it strangely to me. Sorry for the noise. :-(